modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-by-using-the-ocp-web-console_{context}"]
+= Configuring NotificationsConfiguration (CR) by using the {OCP} web console
+
+You can configure `NotificationsConfiguration` (CR) by using the {OCP} web console.
+
+.Prerequisites
+
```

Comment:
Please add the notifications feature should be enabled as a prerequisite.

Go to Argo CD CR -> Click Edit -> YAML 
Under Spec,
Notifications:
 enabled: true

Reply:
Hi @iam-veeramalla, I have provided a link at the end of this procedure in the Additional Resources section for Enabling notifications with an Argo CD Instance. I've also added another line in the procedure. The user can click the link for the section and will be referred to it. Hope this resolves the query.